config/identity/config.yaml

```diff
@@ -236,4 +236,13 @@ ci-issuer-metadata:
   *buildkite-type:
     default-template-values:
       url: "https://buildkite.com"
-    subject-alternative-name-template: "{{.url}}/{{.organization_slug}}/{{.pipeline_slug}}"
+    extension-templates:
+      # Link to the specific Buildkite job that the OIDC token was generated from
+      run-invocation-uri: "{{.url}}/{{.organization_slug}}/{{.pipeline_slug}}/builds/{{.build_number}}#{{.job_id}}"
```

Comment:
The `build_number` claim in Buildkite OIDC tokens is an integer, so this is generating URLs like:

```
https://buildkite.com/yob-opensource/oidc-signing-experiment/builds/%!s(float64=42)#01942921-7883-409b-81d0-3f6b20bcdabf
```

Is there an existing way to have `42` formatted as a string nicely? I had a poke at `pkg/identity/ciprovider/principal.go`, but can't see anything relevant

Reply:
Good find. We can fix this in https://github.com/sigstore/fulcio/blob/main/pkg/identity/ciprovider/principal.go#L33-L39. Either change the formatting string to `%v`, or do type reflection and use `%d` when the type is an integer.